src/Reflex/Class.hs

```diff
@@ -420,6 +420,10 @@ class MonadSample t m => MonadHold t m where
   -- | Create a new 'Event' that only occurs only once, on the first occurrence of
   -- the supplied 'Event'.
   headE :: Event t a -> m (Event t a)
+  -- | An event which occurs at the current moment.
+  now :: m (Event t ())
```

Comment:
Can any laws be documented?

Reply:
I added this if that's what you're thinking of?
```
  -- | An event which only occurs at the current moment in time, such that:
  --
  -- > coincidence (pushAlways (\a -> (a <$) <$> now) e) = e
  --
  now :: m (Event t ())
```
Also, `now` turns events in the context of a `MonadHold` into an applicative functor and monad using the existing `Apply` and `Bind` instances:
```
instance (..., MonadHold t m) => Applicative (m . Event t) where
  pure a = (a <$) <$> now
  (<$>) = liftA2 (<.>)
```